core/src/banking_stage/scheduler_messages.rs

```diff
@@ -0,0 +1,63 @@
+use {
+    crate::immutable_deserialized_packet::ImmutableDeserializedPacket,
+    solana_sdk::transaction::SanitizedTransaction, std::sync::Arc,
+};
+
+/// A unique identifier for a transaction.
+/// The lower 64 bits are based on the order the transaction entered banking stage.
+/// The upper 64 bits are the priority of the transaction.
+#[derive(Clone, Copy, Debug, Default, Hash, PartialEq, Eq, PartialOrd, Ord)]
+pub struct TransactionId(u128);
```

Comment:
Considering a different name. While I use this for uniquely id'ing a transaction, it is also used to sort txs in priority + received order.

Another name option: `PriorityIndex`

Reply:
removed: fe5922dc8523a6ec8941b308cc310e28138522ae